spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/Spark3Util.java

```diff
@@ -448,6 +451,19 @@ public static boolean extensionsEnabled(SparkSession spark) {
     return extensions.contains("IcebergSparkSessionExtensions");
   }
 
+  public static String toIcebergField(String name) {
+    Preconditions.checkArgument(!name.isEmpty(), "Invalid column name: (empty)");
+    if (!isCaseSensitiveEnabled()) {
+      return name.toLowerCase(Locale.ROOT);
+    }
+    return name;
+  }
+
+  public static boolean isCaseSensitiveEnabled() {
+    return Boolean.parseBoolean(
+            SparkSession.active().conf().get("spark.sql.caseSensitive", "false"));
```

Comment:
Nit: Same note about over-indentation. It should be aligned with the second `r` in return.

Reply:
done